server.js

```diff
@@ -72,15 +72,21 @@ bot.on('disconnected', () => {
 });
 
 function onMessage(msg) {
-  if (bot.user.username === msg.author.username) return;
-
   // Checks for PREFIX
   if (msg.content[0] === nconf.get('PREFIX')) {
-    let command = msg.content.toLowerCase().split(' ')[0].substring(1);
-    let suffix = msg.content.substring(command.length + 2);
-    let cmd = commands[command];
+    let cmd;
```

Comment:
You don't need to define this here. It's only used on line 87, best to put it there instead.